### PR TITLE
feat(patterns/pagination): a11y - add aria and role attributes

### LINKED ISSUE #832: Pagination accessibility
# 🚀 Pattern enhancement proposal

### What kind of enhancement are you proposing ?
	
- [ ] On sketch file
- [x] On Documentation
- [x] On source code

### Description

In order to improve accessibility of this component, i purpose some modifications:
- replace the main div by a nav tag with a role="navigation"
- add an attribute aria-label="pagination" on it

### Justification
This is an issue reported in the accessibility audit on the website

**Anything else relevant?**
```
<nav role="navigation" aria-label="pagination" class="mc-pagination" >
   ...
</nav>
```

### PR DESCRIPTION
**Pull request by tiloyi**
Resolves #832

## I have read [the contributing guidelines](https://mozaic.adeo.cloud/Contributing/)

- [x] Yes
- [ ] No

## Does this PR introduce a [breaking change](https://mozaic.adeo.cloud/Contributing/Developers/GitConventions/#breaking-changes-)?

- [ ] Yes
- [x] No

## Describe the changes

**a11y - add aria and role attributes** => Resolves Issue #832

GitHub issue number or Jira issue URL: N/A

### CODE REVIEW

**Review by mohamedMok — APPROVED** [review]

File: src/docs/Components/Pagination/accessibilty.mdx [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+title: "Accessibility"
+order: 3
+---
+
+## Accessibility
+
+In order to guarantee the accessibility of the component, please make sure to respect the following rules when integrating the component into your pages:
```
Comment (mohamedMok) [inline src/docs/Components/Pagination/accessibilty.mdx:8]: In order to guarantee the accessibility of the component => In order to guarantee the component accessibility

File: src/docs/Components/Pagination/accessibilty.mdx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+<HintItem>
+  Always use a <strong>role="navigation"</strong> attribute on the{" "}
```
Comment (mohamedMok) [inline src/docs/Components/Pagination/accessibilty.mdx:11]: maybe remove `{" "}`

File: src/docs/Components/Pagination/accessibilty.mdx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  <strong>.mc-pagination</strong> element.
+</HintItem>
+
+<HintItem>
+  Always use an <strong>aria-label="pagination"</strong> attribute on the{" "}
```
Comment (mohamedMok) [inline src/docs/Components/Pagination/accessibilty.mdx:16]: maybe remove `{" "}`

File: src/docs/Components/Pagination/accessibilty.mdx [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+</HintItem>
+
+```html
+<nav class="mc-pagination" role="navigation" aria-label="pagination">
+  <!-- following code here -->
+</div>
+```
+
+<HintItem>
+  Always use an <strong>aria-label="Previous page"</strong> attribute on the{" "}
```
Comment (mohamedMok) [inline src/docs/Components/Pagination/accessibilty.mdx:27]: maybe remove `{" "}`

File: src/docs/Components/Pagination/accessibilty.mdx [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+  <strong>.mc-pagination__button</strong> element to return to the previous
+  page.
+</HintItem>
+
+<HintItem>
+  Always use an <strong>aria-label="Next Page"</strong> attribute on the{" "}
```
Comment (mohamedMok) [inline src/docs/Components/Pagination/accessibilty.mdx:33]: maybe remove `{" "}`

File: src/docs/Components/Pagination/code.mdx [hunk 6]
```diff
@@ -56,13 +59,14 @@ order: 2
       <use xlink:href="#iconRightL" />
     </svg>
   </a>
-</div>
+</nav>
 ```
 
 <Preview path="pagination-light" />
 
 ## Accessibility and semantic
```
Comment (mohamedMok) [inline src/docs/Components/Pagination/code.mdx:67]: You can remove this part now that you have a tab dedicated

**Review by mohamedMok — APPROVED** [review]
Small changes required.
LGTM otherwise

File: src/docs/Components/Pagination/accessibilty.mdx (hunk 1 above)
Comment (tiloyi) [reply src/docs/Components/Pagination/accessibilty.mdx:8]: Fix

File: src/docs/Components/Pagination/accessibilty.mdx (hunk 2 above)
Comment (tiloyi) [reply src/docs/Components/Pagination/accessibilty.mdx:11]: Fixed

File: src/docs/Components/Pagination/accessibilty.mdx (hunk 3 above)
Comment (tiloyi) [reply src/docs/Components/Pagination/accessibilty.mdx:16]: Fixed

File: src/docs/Components/Pagination/accessibilty.mdx (hunk 4 above)
Comment (tiloyi) [reply src/docs/Components/Pagination/accessibilty.mdx:27]: Fixed

File: src/docs/Components/Pagination/accessibilty.mdx (hunk 5 above)
Comment (tiloyi) [reply src/docs/Components/Pagination/accessibilty.mdx:33]: Fixed

File: src/docs/Components/Pagination/code.mdx (hunk 6 above)
Comment (tiloyi) [reply src/docs/Components/Pagination/code.mdx]: Fixed

**Review by Marine-Fabien — APPROVED** [review]

**Pull request merged**